apps/modernization-ui/src/apps/search/layout/result/NoInput.tsx

```diff
@@ -1,15 +1,17 @@
 import { AlertBanner } from 'apps/page-builder/components/AlertBanner/AlertBanner';
-import styles from './no-patient-result-banner.module.scss';
+import styles from './no-input.module.scss';
 
-export const NoInputBanner = () => {
+const NoInput = () => {
     return (
         <div className={styles.noResults}>
             <AlertBanner type="warning" iconSize={4}>
                 <div className={styles.noResultsContent}>
-                    <span className={styles.noResultsHeader}> No results found</span>
-                    <span className={styles.noResultsSubHeading}>You must enter at least one item to search</span>
+                    <span className={styles.noResultsHeader}>No results found</span>
+                    <span className={styles.noResultsSubHeading}>You must enter at least one item to search.</span>
```

Comment:
Question: Do we not need the punctuation anymore?

Reply:
I had added it to the message after our discussion in stand up that ended with punctuation in headers are not required but should be used in messages